### PR TITLE
5145 create page

### PR DESCRIPTION
**Pull request by samMcCarthyBJSS**
Story [AB#5145](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/5145)
Task [AB#8828](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8828)

image

### CODE REVIEW

File: app/pages/complete-order/contextCreator.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import withFundingManifest from './withFundingManifest.json';
+import { baseUrl } from '../../config';
+
+export const getContext = ({ orderId, orderDescription, fundingSource }) => {
+  const manifest = fundingSource ? withFundingManifest : undefined;
```
Comment (alibombaye) [inline app/pages/complete-order/contextCreator.js:5]: sweet so this is getting it ready for the withoutFundingSource manifest to be provided later?

Comment (samMcCarthyBJSS) [reply app/pages/complete-order/contextCreator.js:5]: yeah exactly thought this was a neat solution for the 2 manifests as me and sam R are pairing on them both

File: app/pages/complete-order/template.njk [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      }) }}
+    </div>
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        {% if (errors.length > 0) %}
+          {{ bcErrorSummary({
+            errors: errors
+          }) }}
+        {% endif %}
```
Comment (alibombaye) [inline app/pages/complete-order/template.njk:21]: Are we ever going to have any errors to show on this page?

Comment (samMcCarthyBJSS) [reply app/pages/complete-order/template.njk:21]: no we won't will remove

File: app/routes.js [hunk 3]
```diff
@@ -80,6 +81,8 @@ export const routes = (authProvider, sessionManager) => {
     res.render('pages/preview/template.njk', addContext({ context, user: req.user }));
   }));
 
+  router.use('/organisation/:orderId/complete-order', completeOrderRoutes(authProvider, addContext, sessionManager));
```
Comment (alibombaye) [inline app/routes.js:84]: oooh nice.. I like this 👍  I think we should do more of this.

**Pull request merged**